kube_aws_autoscaler/main.py

```diff
@@ -129,11 +129,33 @@ def get_nodes_by_asg_zone(autoscaling, nodes: dict) -> dict:
 
     for instance_ids in chunks(list(sorted(instances.keys())), DESCRIBE_AUTO_SCALING_INSTANCES_LIMIT):
         response = autoscaling.describe_auto_scaling_instances(InstanceIds=list(instance_ids))
+
+        asgs = set()
+        for instance in response['AutoScalingInstances']:
+            asgs.add(instance['AutoScalingGroupName'])
+
+        asgresponse = autoscaling.describe_auto_scaling_groups(AutoScalingGroupNames=list(asgs))
+        for asg in asgresponse['AutoScalingGroups']:
+            keep = include_tags is None
+            name = asg['AutoScalingGroupName']
+            for tag in asg['Tags']:
+                if tag['Key'] == 'autoscaled' and tag['Value'] == 'false':
```

Comment:
`autoscaled` != `autoscale` (as in message). I wonder whether this tag name makes sense, it sounds redundant to have an AutoScalingGroup tagged for autoscaling. Maybe prefix the tag key with "kubernetes" or similar?

Reply:
How about kube_aws_autoscaled?  Also, this PR is failing due to needing mocks for the extra aws calls.  Not a python guy myself so I didn't try digging into the tests.